e2e/testdata/violations/most_violations.rego

```diff
@@ -22,6 +22,9 @@ import data.redundant.alias as alias
 # redundant-data-import
 import data
 
+# prefer-package-imports
+import data.rule_named_if.allow
```

Comment:
Worth a comment that this will only fail when there is more than one file linted?

Reply:
Can do 👍